doc/conf.py

```diff
@@ -12,21 +12,20 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import os
-import sys
-from pkg_resources import get_distribution
+from configparser import ConfigParser
+from importlib import metadata, resources
 
 # Get configuration information from setup.cfg
-from configparser import ConfigParser
+pypeit_root = resources.files("pypeit").parent
 conf = ConfigParser()
-
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
+conf.read([pypeit_root / "setup.cfg"])
+setup_cfg = dict(conf.items("metadata"))
```

Comment:
@kbwestfall : Be sure this change is compatible with RTD... I used the same syntax on a package of my own (GH pages rather than RTD) and needed to revert to the `os.path...` version because `resources.files()` pulls the installed package location (somewhere in `site-packages`) rather than the source code location (which includes the desired `setup.cfg` file).

Reply:
And, yes, I see that I was the one who made the change... oops.